src/Tribe/Plugins_API.php

```diff
@@ -0,0 +1,310 @@
+<?php
+
+class Tribe__Plugins_API {
+	/**
+	 * Static Singleton Factory Method
+	 *
+	 * @since tbd
```

Comment:
Aye. I'm woking on the tbd version handling script and plan to have it wrapped up before we package the MR.